Updating "TUTORIAL: How to Get Testnet Funds?"

Jkat opened the pull request:
I'm not sure what's the status of the `sdk-edgenet` and `uat-testnet` and whether or not they're still being maintained. Currently when trying to reach them, I am receiving the following respectively:
```
$ aecli chain status -u https://sdk-edgenet.aepps.com
Request failed with status code 404 undefined
```
```
$ aecli chain status -u https://uat-testnet.aepps.com
getaddrinfo ENOTFOUND uat-testnet.aepps.com uat-testnet.aepps.com:443 undefined
```

Let me know if you'd like me to push another commit to remove them from the tutorial or alternatively to leave them as is for now.

Thanks! ak_25qxcUwYkkYxAhCM7pH8XT5Vow9oowRzERsiYCSZvaU6bTo7Wm

thepiwo commented:
@Jkat as far as I know `sdk-edgenet` and `uat-testnet` are not maintained any more, `sdk-testnet` should be used

alekseytsekov commented on get-testnet-tokens.md:
````diff
@@ -36,7 +38,16 @@ aecli chain status -u https://sdk-testnet.aepps.com
+Solutions_______________________________ 0
+Syncing_________________________________ false
 ```
 ### sdk-edgenet
````
is this network still alive

alekseytsekov commented on get-testnet-tokens.md:
```diff
@@ -88,7 +108,7 @@ There are two ways of getting tokens. The faucets operated by the SDK team will
 - sdk-edgenet - https://edge-faucet.aepps.com/
 
-In the next steps we're going to show you how to get tokens from the faucet for **sdk-edgenet**. The same proces could be followed for the **sdk-testnet**.
+In the next steps we're going to show you how to get tokens from the faucet for **sdk-testnet**. The same proces could be followed for the **sdk-edgenet**.
```
should we still need this `The same proces could be followed for the **sdk-edgenet**.`

thepiwo replied:
@alekseytsekov is edgenet still alive?

Jkat commented:
Thanks for the input, I've updated to remove references to `sdk-edgenet` and `uat-testnet`. I've also cleaned up the wording to reference the singular `sdk-testnet` that's being maintained. Should be ready now.

thepiwo commented:
@alekseytsekov can you approve?

thepiwo commented:
@Pegahbit this PR was created as part of the bounty program as well.
